docs/elasticsearch/reindex-data.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: default
+title: Reindex Data
+parent: Elasticsearch
+nav_order: 3
+---
+
+# Reindex data
+
+After creating an index, if you need to make an extensive change such as adding a new field to every document or performing some computation on all integer fields, rather than deleting your index, making the change offline, and then indexing your data all over again, you can use the `_reindex` operation.
```

Comment:
How common is it to "perform some computation on all integer fields?" Should we go for something more mainstream like combining multiple indices, or is this a good use case?